sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala

```diff
@@ -145,7 +146,8 @@ object Webpack {
       .getOrElse(generatedWebpackConfigFile.file)
 
     log.info("Bundling the application with its NPM dependencies")
-    Webpack.run("--config", configFile.absolutePath)(targetDir, log)
+    val args = extraArgs ++: Seq("-config", configFile.absolutePath)
```

Comment:
I guess there is a typo here, there should be two dashes before “config”, right?

Reply:
Fixed now, I forgot to upload the fix